.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: CI Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy: 
+      matrix : 
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Do we care about any other platforms? macOS?

Reply:
Good point, added